setup.cfg

```diff
@@ -1,10 +1,68 @@
 [metadata]
-package_name = jwst
-description = JWST Pipeline
-author = STScI
-license = BSD
-edit_on_github = False
-github_project = spacetelescope/jwst
+name = jwst
+description = Library for calibration of science observations from the James Webb Space Telescope
+author = JWST calibration pipeline developers
+license = BSD 3-Clause License
```

Comment:
Should this be the "official" short identifier?  Only asking because that turned up as a warning when I last submitted an update to asdf on one of the conda channels.  Per https://spdx.org/licenses/BSD-3-Clause.html, `BSD-3-Clause` is the short identifier for this license.

Reply:
Ah, thanks!

We are not delivering to any conda channels, but it would be good to standardize this.  That said, this was my example

https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files

💭

Reply:
I've updated to `BSD-3-Clause`.